api/controllers/users.py

```diff
@@ -34,6 +35,19 @@ def put(self, user_email):
             'user_email': user_id
         }
 
+    @api.doc('delete_user')
+    @api.expect(delete_users_model)
+    @jwt_required
+    @check_permissions()
+    def delete(self, user_email):
+        '''Deletes a user given its email'''
+        deleted = users_service.delete_user(user_email)
+        if deleted:
+            return {
+                'user_email': user_email
+            }
+        return { 'error': 'Unable to delete user' }
```

Comment:
What are your thoughts on raising an exception vs returning error? If we return an  error object, it will have a 200 HTTP status code. Raising exception will have 500. Another option is to do `return { 'error' : 'Unable to delete user'}, 500` which I think will give it a 500 status code. I just want to make sure we know when this fails as it can have pretty catastrophic consequences if part of a user gets deleted but then another part doesnt

Reply:
Yeah good point I'll just change it to raise an exception